api/login/login.go

```diff
@@ -0,0 +1,15 @@
+package login
+
+import (
+	"github.com/brooklyncentral/brooklyn-cli/error_handler"
+	"github.com/brooklyncentral/brooklyn-cli/net"
+)
+
+func Login(network *net.Network) string {
```

Comment:
I wasn't quite sure what would end up in the package at the start.  But since it is now effectively a duplicate of version I will remove this to avoid the duplication.